.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41E Bug report"
+about: Let us know if something isn't working as you expect it to.
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**WordCamp:**
+If this is a problem on a specific WordCamp's site, list that here.
+
+**System (please complete the following information):**
+ - Device: [e.g. laptop, iPhone6]
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Additional context**
+- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.
```

Comment:
`MAYBE` I'd move this to the top, and maybe make it an HTML comment so people don't leave it in the report through laziness.

Maybe change the `additional context` heading to `Security Issues` or something, so it's more descriptive.